src/process/linux.rs

```diff
@@ -63,6 +63,8 @@ impl ProcessTask {
 pub struct ProcessInfo {
     pub pid: i32,
     pub ppid: i32,
+    pub pgid: i32,
+    pub session: i32,
```

Comment:
I think this is not required because `pgid` and `session` can be taken through `curr_proc`.

Reply:
You're right, fixed